wradlib/georef/raster.py

```diff
@@ -27,37 +27,40 @@
 from osgeo import gdal, osr, gdal_array
 
 from .projection import reproject
+from ..io import dem
 
 
-def pixel_coordinates(nx, ny, mode="centers"):
-    """Get pixel coordinates from a regular grid with dimension nx by ny.
+def pixel_coordinates(nx, ny, edge=False):
+    """Get pixel center coordinates from a regular grid with dimension nx by ny.
 
     Parameters
     ----------
     nx : int
         xsize
     ny : int
         ysize
-    mode : string
-        `centers` or `edges` to return the pixel coordinates
-        defaults to centers
+    edge : bool
```

Comment:
Might just leave the string kwarg as it is.

Reply:
Yes indeed